app/templates/resources/add.html

```diff
@@ -0,0 +1,133 @@
+                'error':   get_flashed_messages(category_filter=['form-error']),
+                'warning': get_flashed_messages(category_filter=['form-check-email']),
+                'info':    get_flashed_messages(category_filter=['form-info']),
+                'success': get_flashed_messages(category_filter=['form-success'])
+            } %}
+
+            {{ f.begin_form(form, flashes) }}
+
+            <div class="two fields">
+                {{ f.render_form_field(form.name) }}
+                {{ f.render_form_field(form.website) }}
+            </div>
+
+            {{ f.render_form_field(form.description) }}
+
+            {{ f.render_form_field(form.autocomplete) }}
+
+            <div class="two fields">
+                {{ f.render_form_field(form.street_number) }}
+                {{ f.render_form_field(form.route) }}
+            </div>
+
+            {{ f.render_form_field(form.locality) }}
+
+            <div class="two fields">
+                {{ f.render_form_field(form.administrative_area_level_1) }}
+                {{ f.render_form_field(form.postal_code) }}
+            </div>
+
+            {{ f.form_message(flashes['error'], header='Something went wrong.', class='error') }}
+            {{ f.form_message(flashes['warning'], header='Check your email.', class='warning') }}
+            {{ f.form_message(flashes['info'], header='Information', class='info') }}
+            {{ f.form_message(flashes['success'], header='Success!', class='success') }}
+
+            {% for field in form | selectattr('type', 'equalto', 'SubmitField') %}
+                {{ f.render_form_field(field) }}
+            {% endfor %}
+
+            {{ f.end_form(form) }}
+        </div>
+    </div>
+
+<script>
+// Set onFocus attribute for autocomplete field and disabled attribute for address fields
+$( document ).ready(function() {
+  $("#autocomplete").focus(geolocate())
+
+  for (var component in componentForm) {
+    document.getElementById(component).disabled = true;
+  }
+})
+
+
+// https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
+// This example displays an address form, using the autocomplete feature
+// of the Google Places API to help users fill in the information.
+
+var placeSearch, autocomplete;
+var componentForm = {
+  street_number: 'short_name',
+  route: 'long_name',
+  locality: 'short_name',
+  administrative_area_level_1: 'short_name',
+  postal_code: 'short_name'
+};
+
+function initAutocomplete() {
+  // Create the autocomplete object, restricting the search to geographical
+  // location types.
+  autocomplete = new google.maps.places.Autocomplete(
+      /** @type {!HTMLInputElement} */(document.getElementById('autocomplete')),
+      {types: ['geocode']});
+
+  // When the user selects an address from the dropdown, populate the address
+  // fields in the form.
+  autocomplete.addListener('place_changed', fillInAddress);
+}
+
+// [START region_fillform]
```

Comment:
@weronica copied from Google Place Autocomplete, beginning of filling in address fields once autocompleting is done (See https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform).